### ISSUE TITLE
Please somebody post Netgear a6210 iw list info 

### ISSUE THREAD

**Issue by Genxster1998** [morrownr/7612u#10]
Can we increase txpower of Netgear a6210 from 18dbm to anything else ?
It's stuck at 18dbm for all frequencies and channels ...
Do anybody have any info ?
Is it due to country reg info ?
@morrownr please help .
I am thinking to replace it's antenna to 5dbm external with rpsma connector .I have opened it ,it has ipex connector which should work .

**Comment by morrownr** [comment]
Hi @Genxster1998 

https://github.com/morrownr/USB-WiFi/tree/main/iw_list

> It's stuck at 18dbm for all frequencies and channels ...

I have one but have not investigated this. A lot of adapters these days have txpower locked in firmware. I don't know this is the case here. What is it that you are trying to do?

Regards

**Comment by morrownr** [comment]
> It's stuck at 18dbm for all frequencies and channels ...

Forgot to mention this: I don't pay much attention to txpower. The longest range adapter that I have, the ALFA ACHM, shows 2dBm. Yep, that is 2 as in one plus one. I also have an ALFA ACH which has very good range and is second only to the ACHM in that regards. I just looked... 20 dBm.

Here is a good story to read:

https://metis.fi/en/2017/10/txpower/

Regards

**Comment by Genxster1998** [comment]
@morrownr Ohh so it's 20dBm for A6210 for you , right !
I can reduce it but not increase it beyond 18dBm , any patch in firmware that I can do ?
You are more than satisfied with Achm range but for me this high priced thing perform worse(low signal and speed) than my Laptop Internal WiFi adapter (Atheros Card)
Thats why I am looking to increase txpower since 500mw ~ 20dbm should not be bottleneck .
I have tried Atheros 9565 , Realtek 8812BU (D-Link  dwa-182) , MT7612* (Netgear A6210)
Best performer is indeed pcie adapter Atheros ...
Rest I can't seem to differentiate between mt7612u and rtl8812bu ? I found mt76 just much hyped .

**Comment by morrownr** [comment]
>  Ohh so it's 20dBm for A6210 for you , right ?

No. I haven't checked. If the A6210 shows 18 dBm for you then it is likely going to show 18 dBm for me.

> You are more than satisfied with Achm range but for me this high priced thing perform worse(low signal and speed) than my Laptop Internal WiFi adapter (Atheros Card)

What high priced thing are you talking about?

> Thats why I am looking to increase txpower

There are various reasons why txpower is limited on some adapters, 

> Rest I can't seem to differentiate between mt7612u and rtl8812bu ?

Generlly speaking, the performance capabilities of the 7612u and 8812bu are about the same so I would expect approximately the same performance if all else like antenna capability is similar when comparing adapters. However, the difference comes in with several capabilities that the 7612u driver has the 8812bu driver does not. 7612u based adapters will be less hassle over time also.

>  I found mt76 just much hyped .

Please point me to this hype you speak of. It sounds to me that you have bought an adapter that is not appropriate for your needs. There are usb wifi adapters that come in many sizes, speeds and capabilities. Getting the right adapter makes all the difference in the world. What capabilities are you look for?

Before you answer... the above will be my last reply if you come back again with nothing but complaining, On the other hand, if you are looking to increase your knowledge, that I will help you with.

**Comment by Genxster1998** [comment]
> > Ohh so it's 20dBm for A6210 for you , right ?
> 
> No. I haven't checked. If the A6210 shows 18 dBm for you then it is likely going to show 18 dBm for me.
Okay
> > You are more than satisfied with Achm range but for me this high priced thing perform worse(low signal and speed) than my Laptop Internal WiFi adapter (Atheros Card)
> 
> What high priced thing are you talking about?
Netgear A6210 than others which have same chipset/competitive chipset with better range at much low cost.
> > Thats why I am looking to increase txpower
> 
> There are various reasons why txpower is limited on some adapters,
 This is self understanding ...
> > Rest I can't seem to differentiate between mt7612u and rtl8812bu ?
> 
> Generlly speaking, the performance capabilities of the 7612u and 8812bu are about the same so I would expect approximately the same performance if all else like antenna capability is similar when comparing adapters. However, the difference comes in with several capabilities that the 7612u driver has the 8812bu driver does not. 7612u based adapters will be less hassle over time also.
yeah! concurrent_mode right for evil twins etc.
> > I found mt76 just much hyped .
> 
>> Please point me to this hype you speak of. It sounds to me that you have bought an adapter that is not appropriate for your needs. There are usb wifi adapters that come in many sizes, speeds and capabilities. Getting the right adapter makes all the difference in the world. What capabilities are you look for?
hype means like recomendation from your blog . 
> Before you answer... the above will be my last reply if you come back again with nothing but complaining, On the other hand, if you are looking to increase your knowledge, that I will help you with.
I am not complaining to you about anything just looking to solve problem as neither you manufacture or endorse it commercially , I am just here to tweak it if it could be as being oss at mainline kernel ...
Rest is all just my opinion .

**Comment by morrownr** [comment]
> hype means like recommendation from your blog .

verb (used with object), hyped, hyp·ing.
to stimulate, excite, or agitate (usually followed by up):
She was hyped up at the thought of owning her own car.
to create interest in by flamboyant or dramatic methods; promote or publicize showily:
a promoter who knows how to hype a prizefight.
to intensify (advertising, promotion, or publicity) by ingenious or questionable claims, methods, etc. (usually followed by up).
to trick; gull.

noun
exaggerated publicity; hoopla.
an ingenious or questionable claim, method, etc., used in advertising, promotion, or publicity to intensify the effect.

I do not care for your use of the word "hype" as the word has a very negative connotation. I am not hyping anything. Nor am I making any money from this site.

You appear to have bought the wrong adapter, My review of this adapter makes it quite clear that the adapter has a limited range.

Good luck with your problem.
